frontend/src/app/testResults/uploads/Uploads.scss

```diff
@@ -0,0 +1,5 @@
+.sr-test-results-uploads {
+  .usa-process-list__heading {
+    font-size: 1.17rem;
```

Comment:
⚠️ The current font-size is `18.75px` in rem that is `1.171875rem` but I just set it to `1.17rem` let me know if you want this to be more precise.